actions.yaml

```diff
@@ -5,6 +5,9 @@ force-refresh-start:
     Force upgrade of this unit.
     Potential of data loss and downtime.
 
+pre-refresh-check:
```

Comment:
question - will this now be a required action to be ran on mongos charms?

Reply:
It will still be run before upgrading units but yes, it's been added for consistency between all charms.
It can be removed but since it's there on k8s already, I think it's good that it's everywhere the same.

Reply:
I think we should drop, as to not mislead the users that everything is "fine", unless we actually performed refresh checks

Reply:
It does perform some refresh checks!
The exact checks that are automatically run before upgrading, in this case it fails if mongos is not able to read write before the ugprade.